packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -250,7 +250,7 @@ export default function LabeledField(props: Props) {
             ]
                 .filter(Boolean)
                 .join(" "),
-            required: isRequired,
```

Comment:
Since we were passing in a boolean, the default required error message was always used 😅